lib/routes/client-api/feature.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { Router } = require('express');
+const commonFeatureToggle = require('../common/feature.js');
+
+exports.router = config => {
+    const router = Router(); // eslint-disable-line new-cap
+
+    router.get('/', commonFeatureToggle.router(config));
```

Comment:
Actually this will not be sufficient, we have "non-official" clients (ha-proxy-integration) fetching the actual toggles directly (`/features/:toggleName`)